src/utils.js

```diff
@@ -7,3 +7,7 @@ export function loadFromLocal(name) {
   const data = JSON.parse(jsonString)
   return data
 }
+
+export function loadActivities() {
+  loadFromLocal('activities')
```

Comment:
Maybe put the save string in a const and do the same simplification for your save function?